var/spack/repos/builtin/packages/binutils/package.py

```diff
@@ -76,6 +76,8 @@ def install(self, spec, prefix):
 
         if '+libiberty' in spec:
             configure_args.append('--enable-install-libiberty')
+            configure_args.append('--disable-nls')
```

Comment:
Thanks! I think it will be much easier to get this merged if it's two separate PRs. The homepage changes are pretty easy to approve. Changes to binutils may take longer to approve.